### PR TITLE
updated frontend to use event instead of class

### PR DESCRIPTION
**Pull request by gateship1**
associated with backend pull request #88

### CODE REVIEW

File: src/Components/ViewEvent.jsx [hunk 1]
```diff
@@ -22,23 +22,23 @@ class ViewEvent extends Component {
...
-      classId: '',
+      eventId: '',
       className: '',
       meetingLink: '',
       year: '',
       section: ''
     };
 
-    this.changeClassId = this.changeClassId.bind(this);
+    this.changeeventId = this.changeeventId.bind(this);
```
Comment (msknobloch2) [inline src/Components/ViewEvent.jsx:32]: capitalize Event

File: src/Components/ViewEvent.jsx [hunk 2]
```diff
@@ -22,23 +22,23 @@ class ViewEvent extends Component {
...
     this.changeClassName = this.changeClassName.bind(this);
     this.changeYear = this.changeYear.bind(this);
     this.changeSection = this.changeSection.bind(this);
     this.onFormSubmitted = this.onFormSubmitted.bind(this);
     this.env = new Environment();
   }
 
-  changeClassId(event) {
-    this.setState({ classId: event.target.value });
+  changeeventId(event) {
```
Comment (msknobloch2) [inline src/Components/ViewEvent.jsx:40]: capitalize Event

File: src/Components/ViewEvent.jsx [hunk 3]
```diff
@@ -152,9 +152,9 @@ class ViewEvent extends Component {
             </Row>
             <Row className="justify-content-md-center">
               <Col xs={12} md={5}>
-                <Form.Group controlId="classIdFormGroup">
+                <Form.Group controlId="eventIdFormGroup">
                   <Form.Label className="rowStyle">Class Id:</Form.Label>
-                  <Form.Control type="text" name="classIdInput" disabled={this.state.disabled} value={this.state.classId} onChange={this.changeClassId} />
+                  <Form.Control type="text" name="eventIdInput" disabled={this.state.disabled} value={this.state.eventId} onChange={this.changeeventId} />
```
Comment (msknobloch2) [inline src/Components/ViewEvent.jsx:157]: capitalize Event

**Review by msknobloch2 — CHANGES_REQUESTED** [review]

File: src/Components/ViewEvent.jsx (hunk 3 above)
Comment (gateship1) [reply src/Components/ViewEvent.jsx]: accepted

File: src/Components/ViewEvent.jsx (hunk 2 above)
Comment (gateship1) [reply src/Components/ViewEvent.jsx]: accepted

File: src/Components/ViewEvent.jsx (hunk 1 above)
Comment (gateship1) [reply src/Components/ViewEvent.jsx]: accepted

**Review by msknobloch2 — CHANGES_REQUESTED** [review]
i think you made it worse

**Review by msknobloch2 — APPROVED** [review]

**Review by coxwm11 — APPROVED** [review]

**Review by jonathanmcdaniel — APPROVED** [review]

**Review by jdurelli — APPROVED** [review]

**Pull request merged**